src/zjs_gpio.c

```diff
@@ -325,12 +379,12 @@ static jerry_value_t zjs_gpio_open(const jerry_value_t function_obj,
         // Zephyr ISR callback init
```

Comment:
@daniel-thompson : I of course noticed matters which you talk about, but they definitely aren't related to the topic of this patch. This patch tries to support GPIOs of any Zephyr board. You're talking about optimizing how the module works. That's definitely useful to consider separately, though I'd call a "premature optimization" cliche (as there're talks about switching to completely different GPIO, etc. APIs).